collect_app/src/androidTest/java/org/odk/collect/android/espressoutils/pages/Page.java

```diff
@@ -72,6 +72,17 @@ public T checkIsTextDisplayed(String text) {
         return (T) this;
     }
 
+    public T checkIsTextDisplayed(String...  text) {
+        for (String s : text) {
+            try {
+                onView(withText(s)).check(matches(isDisplayed()));
+            } catch (NoMatchingViewException e) {
+                Timber.i(e);
```

Comment:
This means the test won't fail if it can't find one of the items passed in right? Maybe we could rename the method to something like `checkIsTextDisplayedLoosley` and add a comment to make sure that's clear.

Reply:
It should fail, my mistake, fixed.

Reply:
👍